### DISCUSSION TITLE
How to stop language server client from ordering result?

### DISCUSSION THREAD

**Discussion by JustFrederik** [zed-industries/zed#11205] [Feature Requests]
Im trying to setup a lsp which orders the results. It's a crate suggestion feature. If the name contains only a single version(0.1.0) or the name contains a number it's unlikely that this is what is searched for and therefore needs to moved down. My ordering isn't kept. What can I change to order allow server ordering?

<img width="718" alt="Screenshot 2024-04-30 at 09 35 44" src="https://github.com/zed-industries/zed/assets/121750393/27bd6d09-c836-4614-83cb-88ba5f10d823">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
